src/audio/module_adapter/module_adapter.c

```diff
@@ -281,6 +281,7 @@ static int module_adapter_dp_queue_prepare(struct comp_dev *dev)
 		i++;
 	}
 	mod->num_of_sources = i;
+	uint32_t period = UINT32_MAX;
```

Comment:
can we use `unsigned int period = MAX_UINT;` and the same type for `sink_period` below?